README.md

```diff
@@ -15,41 +15,60 @@ resources:
       endpoint: dirigeants # This is likely your github username or organization name
 ```
 
-**Note:** You will have to allow your pipeline to use your github credentials to be used to fetch templates from other repos.
-You can do this by going to your Project Settings (in azure devops):
+> **Note:** You will have to allow your pipeline to use your github credentials to be used to fetch templates from other
+repos.
+
+You can do this by going to your Project Settings (in Azure DevOps):
 1. Pipelines
-1. Service Connections 
-1. (your username/orgname should be listed, otherwise the endpoint will be whatever is listed) 
-1. Policies 
-1. Check: `Allow all pipelines to use this endpoint` 
+1. Service Connections
+1. (Your username/organization name should be listed, otherwise the endpoint will be whatever is listed)
+1. Policies
+1. Check: `Allow all pipelines to use this endpoint`
 1. Save Changes
 
 ### Lint Template
 
-This template will install the specified version of node.js, install dependencies, and run the `test:lint` script defined in your package.json. (meaning you can use any lint tool or any combination of tools)
+This template will install the specified version of Node.js, install dependencies, and run the `test:lint` script
+defined in your `package.json` (meaning you can use any linter or any combination of tools).
 
 ```yml
 jobs:
   - template: lint.yml@templates
-    parameters: # Optional if you want to use all defaults
-      versionSpec: '10.x' # Optional (default: '10.x'): The version of node.js you want to use, valid values are ['8.x', '9.x', '10.x', '11.x', ...]
+    # Optional: Whether or not you want to use all defaults
```

Comment:
the original text is more accurate, it's not optional if you do not want to use all defaults

Reply:
Reverted in 07c583d